### PR TITLE
Add allure output to generate_object about storage object to container.

### PR DESCRIPTION
**Pull request by vvarg229**
It helps in parsing failed tests.

### CODE REVIEW

File: pytest_tests/helpers/container.py [hunk 1]
```diff
@@ -74,14 +74,14 @@ def generate_object(
...
-
-            storage_object = StorageObjectInfo(
-                container_id,
-                object_id,
-                size=size,
-                wallet_file_path=wallet_path,
-                file_path=file_path,
-                file_hash=file_hash,
-            )
+            with allure.step(f"Storage object with size {size} to container {container_id}"):
```
Comment (roman-khimov) [inline pytest_tests/helpers/container.py:77]: _store_ object of size to?

**Review by roman-khimov — COMMENTED** [review]
Please also use "component:" prefix for commit messages if possible.

**Comment by vvarg229** [comment]
> Please also use "component:" prefix for commit messages if possible.

Thanks. 
In neofs-testcases project, previous commits were made without a "component:" prefix.
But it seems like a really good idea.

File: pytest_tests/helpers/container.py (hunk 1 above)
Comment (vvarg229) [reply pytest_tests/helpers/container.py]: Fixed

**Pull request merged**